webpack.config.js

```diff
@@ -19,6 +19,8 @@ var wpconfig = {
 
     entry: {
         // Studio
+        AssetsPage: './node_modules/@edx/studio-frontend/src/AssetsPage/index.jsx',
+        // if not-dev: AssetsPage: './node_modules/@edx/studio-frontend/dist/assets.min.js'
```

Comment:
`isProd` is defined above, no clue how to do conditionals in the `entry` list though.